opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -503,7 +503,7 @@ def get_current_span(self):
     def start_as_current_span(
         self,
         name: str,
-        parent: trace_api.ParentSpan = trace_api.Tracer.CURRENT_SPAN,
+        parent: trace_api.Parent = trace_api.Tracer.CURRENT_SPAN,
```

Comment:
ditto here, shouldn't these be a SpanContext?

Reply:
no, as the spec specifies all Trace API methods as accepting either Span / SpanContext.